index.bs

```diff
@@ -12394,12 +12394,7 @@ more work than is possible in real-time given the CPU's speed.
 <h2 id="priv-sec">
 Security and Privacy Considerations</h2>
 
-The W3C TAG is developing a <a href="https://w3ctag.github.io/security-questionnaire/">Self-Review
-Questionnaire: Security and Privacy</a> for editors of specifications
-to informatively answer.
-
```

Comment:
The above paragraph isn't really needed anymore since TAG has developed the questionnaire.